test/unit/provisioning/device-configGroup-api-test.js

```diff
@@ -0,0 +1,1022 @@
+/*
```

Comment:
Is test/unit/provisioning/device-configGroup-api-test.js a "parallel" of some existing service-based test?

Reply:
Yes, It is parallel of test/unit/provisioning/device-group-api-test.js